react/week4/gitrepos/src/contexts/GlobalStateContext.js

```diff
@@ -0,0 +1,2 @@
+import { createContext } from "react";
+export const GlobalStateContext = createContext();
```

Comment:
I'd recommend have this and the Provider context in the same file. As your app grows and you have many contexts then you're creating x2 the amount of files.

Reply:
It took me a while to figure out how to separate them.
My thought was, context belongs to the context folder, and the GlobalStateContextProvider belongs to components, since I created this component and I was not using the GlobalStateContext. Provider.
But thank you for the advice, i'm going to put them in the same folder from now on!